app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BridgeServiceManager.kt

```diff
@@ -0,0 +1,14 @@
+package com.x8bit.bitwarden.data.platform.manager
+
+import com.bitwarden.bridge.IBridgeService
+
+/**
+ * Provides access to [IBridgeService] APIs in an injectable and testable manner.
+ */
+interface BridgeServiceManager {
```

Comment:
Can we call this a `Processor` instead of manager.

Reply:
Sure thing!

2590169fc877c149894b6c7570917ee2dd0e6534